src/client/api/quoteSelector.ts

```diff
@@ -0,0 +1,63 @@
+import { InsuranceType } from 'utils/hooks/useSelectedInsuranceTypes'
+import { BundledQuote, ApartmentType } from 'data/graphql'
+
```

Comment:
Maybe these functions would be more reusable if instead of a `quote` they just took `data` directly? Then they can be used even when we don't have access to the entire quote, but only the `data` object 🤔

Reply:
Hmm do you have an example of such a use case? We are trying to find the right abstration-level.

Our best hunch now is to provide selectors for "bundle" and "quote".

Reply:
I think there will be at least one case, in the "Summary"/"Your home" section on the checkout details page, where we only have access to the `data` object(s), but still want to be able to check for example if it's a student quote.

Reply:
But we can change that when we get there :)